api/tests/connectors/titelive/titelive_test.py

```diff
@@ -54,6 +55,8 @@ def test_get_new_product_from_ean_13(self, requests_mock):
         assert product.thumbCount == article["image"]
         assert product.name == oeuvre["titre"]
         assert product.subcategoryId == subcategories.LIVRE_PAPIER.id
+        assert product.lastProvider is not None
+        assert product.lastProvider.name == providers_constants.TITELIVE_EPAGINE_PROVIDER_NAME
```

Comment:
[micro-détail] On n'a peut-être pas besoin de la première vérification, puisqu'elle est "inclue" dans la seconde.

```suggestion
        assert product.lastProvider.name == providers_constants.TITELIVE_EPAGINE_PROVIDER_NAME
```